src/modules/device.json

```diff
@@ -268,17 +277,23 @@
             "value": {
               "sdk": {
                 "major": 0,
-                "minor": 5,
+                "minor": 8,
                 "patch": 0,
-                "readable": "Firebolt JS SDK v0.5.0"
+                "readable": "Firebolt JS SDK v0.8.0"
```

Comment:
This is test code that has to be manually kept in sync with the real `package.json` file. That's no good because it can be manipulated by automation tasks. Instead of testing real package.json, I would recommend that this test some unchanging mock of the `package.json` file. If that isn't satisfactory, then we are testing something besides just that code somewhere is correctly interpolating values from a file.

Reply:
Okay, good plan, but don't want to hold up this feature. will create a todo.